src/AppBundle/Entity/Client.php

```diff
@@ -44,14 +44,14 @@ class Client
      *
      * @JMS\Groups({"client-reports"})
      * @JMS\Type("array")
-     * @ORM\OneToMany(targetEntity="AppBundle\Entity\Report\Report", mappedBy="client", cascade={"persist"})
+     * @ORM\OneToMany(targetEntity="AppBundle\Entity\Report\Report", mappedBy="client", cascade={"persist", "remove"})
```

Comment:
Dont we had a check for this within admin. Are we sure we want to do this? :)

Reply:
there is a db cascade delete  on client.report, and this is corresponding application-level cascade (only useful to keep the unity of work clean, as the cascade delete logic is already applied by the db). I needed this cascade to delete clients without users